### PR TITLE
Make the code build

### PR DESCRIPTION
**Pull request by sbose78**
```
$ dep ensure -v
$ go build -v -o ./bin/devopsconsole-operator cmd/manager/main.go
```

### CODE REVIEW

File: deploy/crds/devconsole_v1alpha1_envdeployment_crd.yaml [hunk 1]
```diff
@@ -1,9 +1,9 @@
 apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
-  name: envdeployments.devconsole.openshift.io
+  name: envdeployments.devopsconsole-operator.openshift.io
 spec:
-  group: devconsole.openshift.io
+  group: devopsconsole-operator.openshift.io
```
Comment (alexeykazakov) [inline deploy/crds/devconsole_v1alpha1_envdeployment_crd.yaml]: I think we should use `devopsconsole.openshift.io` group for all our DevOps Console APIs. Not 'devopsconsole-operator.openshift.io'.

Comment (sbose78) [reply deploy/crds/devconsole_v1alpha1_envdeployment_crd.yaml]: Ah right! https://github.com/redhat-developer/devopsconsole-operator/pull/4/commits/d65f2374124b477a156a669c966561b240929bc3

Comment (alexeykazakov) [reply deploy/crds/devconsole_v1alpha1_envdeployment_crd.yaml]: Looks good. What about the other places too? `devopsconsole-operator.openshift.io` group is all over the place :)

Comment (sbose78) [reply deploy/crds/devconsole_v1alpha1_envdeployment_crd.yaml]: Right https://github.com/redhat-developer/devopsconsole-operator/pull/4/commits/cf023c28b7b7cb31d4302a7e5c7f02da151ae9c0

**Pull request merged**